src/frontend/Komponenter/Journalforing/LeggTilBarnSomSkalFødes.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { FamilieDatovelger } from '@navikt/familie-form-elements';
+import LeggtilMedSirkel from '../../Felles/Ikoner/LeggtilMedSirkel';
+import SlettSøppelkasse from '../../Felles/Ikoner/SlettSøppelkasse';
+import { BarnSomSkalFødes } from '../../App/hooks/useJournalføringState';
+import { Button, Heading } from '@navikt/ds-react';
+import styled from 'styled-components';
+import navFarger from 'nav-frontend-core';
+
+const Tittel = styled(Heading)`
+    color: ${navFarger.navBlaLighten40};
+`;
+
+const LeggTilBarnContent = styled.div`
+    margin: 0.5rem 0;
+`;
+
+const InlineContent = styled.div`
+    border-left: 2px solid ${navFarger.navBlaLighten40};
+    padding-left: 1rem;
+    margin-left: 1rem;
+`;
+
+const TerminbarnMedDatovelger = styled.div`
+    display: grid;
+    grid-auto-columns: minmax(auto, 12rem);
+    grid-auto-flow: column;
+
+    > * {
+        align-self: flex-end;
+        padding: 0.5rem 0.5rem;
+    }
+`;
+
+const FjernBanrKnapp = styled(Button)`
+    width: 4rem;
+`;
+const LeggTilBarnKnapp = styled(Button)`
+    margin-top: 1rem;
+`;
+
+const LeggTilBarnSomSkalFødes: React.FC<{
+    barnSomSkalFødes: BarnSomSkalFødes[];
+    oppdaterBarnSomSkalFødes: (terminbarn: BarnSomSkalFødes[]) => void;
+}> = ({ barnSomSkalFødes, oppdaterBarnSomSkalFødes }) => {
+    const leggTilBarn = () => oppdaterBarnSomSkalFødes([...barnSomSkalFødes, {}]);
+
+    const fjernBarn = (terminbarnSomSkalSlettes: BarnSomSkalFødes) =>
+        oppdaterBarnSomSkalFødes(
+            barnSomSkalFødes.filter((barn) => barn !== terminbarnSomSkalSlettes)
+        );
+
+    const oppdaterTermindato = (terminbarnSomSkalOppdateres: BarnSomSkalFødes, dato: string) =>
+        oppdaterBarnSomSkalFødes(
+            barnSomSkalFødes.map((barn) =>
+                barn === terminbarnSomSkalOppdateres ? { fødselTerminDato: dato as string } : barn
```

Comment:
Funker denne, eller trenger du en uuid per barn for å finne riktig barn som skal oppdateres? (tilsvarende ovenfor for sletting...)

Reply:
Blir kanskje litt kokos på tvillinger?